WebDriverAgentLib/Utilities/XCUIApplicationProcessDelay.h

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import <Foundation/Foundation.h>
+#import <CoreGraphics/CoreGraphics.h>
```

Comment:
is this import needed here?

Reply:
This header should actually be deleted already. It's not in the tree of my branch anymore, I'll merge this now and lets see if it's gone...

Reply:
Oh, I can't merge... someone else has to 😄

Reply:
maybe you didn't push your recent changes?